libmscore/instrument.cpp

```diff
@@ -428,7 +427,7 @@ const char *Channel::DEFAULT_NAME = "normal";
 Channel::Channel()
       {
       for(int i = 0; i < int(A::INIT_COUNT); ++i)
-            init.push_back(MidiCoreEvent());
+            initList().push_back(MidiCoreEvent());
```

Comment:
It looks like it is better to use init list directly here rather than via an access function as this is just an initialization of this list. This wouldn't make a difference currently but it may have an effect in case other possible changes I mention here are made.

Reply:
fixed